src/scenes/Wallet/components/History/components/HistoryItem/index.js

```diff
@@ -64,3 +64,5 @@ const SecondaryRow = withStyles(styles)(({ classes, transaction, expanded }) =>
     <TableCell /><TableCell /><TableCell /><TableCell />
   </TableRow>
 ));
+
+const moneyNum = (num) => (Math.floor(Number(num) * 100000) / 100000).toFixed(6); // eslint-disable-line
```

Comment:
It seems like this truncate the number to 5th decimal place but fixed at 6th decimal place. (3.1415926535 => 3.14150).

Reply:
updated